bstats-bukkit-lite/pom.xml

```diff
@@ -70,9 +23,9 @@
         <dependency>
             <groupId>org.bukkit</groupId>
             <artifactId>bukkit</artifactId>
-            <version>1.12.2-R0.1-SNAPSHOT</version>
+            <version>1.13.2-R0.1-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
     </dependencies>
 
-</project>
+</project>
```

Comment:
^ You've discarded the newline-at-end-of-file from quite a few files in this PR. As you can see by the little red symbol here, GitHub renders this explicitly as a mistake (per general code practice, all code files should end with a newline unless there's a very specific reason to not have that).

Reply:
Interestingly enough, my GitHub doesn't want to show that symbol. Weird

Reply:
It only displays on some versions of the page (I don't know what logic GitHub uses for it), mainly the full-commit view (or full-PR-change view) show it. Just for reference, this is how GitHub shows it:
![No-newline-at-EOL](https://i.alexgoodwin.media/i/misc/cdec0c.png)

Reply:
The majority of the files in the projects doesn't use newline at the end of the file, i just uniformed the codestyle.

Reply:
Yeah, i'm a Newline-at-EOF guy, but i was concerned bastian had a different opinion on that